srcpkgs/python3-networkx/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'python3-networkx'
 pkgname=python3-networkx
-version=2.6.2
+version=2.6.3
 revision=1
 wrksrc="networkx-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools unzip"
```

Comment:
The unzip dependency seems to be not be necessary since 03a764245856b205c395f4bbb379cca12455409f

Reply:
OK. I have removed this.